pgmpy/models/BayesianModel.py

```diff
@@ -1030,3 +1030,16 @@ def get_markov_blanket(self, node):
         blanket_nodes = set(blanket_nodes)
         blanket_nodes.remove(node)
         return list(blanket_nodes)
+    
+    def __eq__(self, other):
+        if isinstance(other, self.__class__):
+            network_struct1 = (self.nodes(),self.edges())
+            network_struct2 = (other.nodes(),other.edges())
+            cpd1 = self.cpds
+            cpd2 = other.cpds
```

Comment:
Never use names with `cpd1` and `cpd2`. Come up with better variable names which explains what they are.